### PR TITLE
Glyph3d  to Glyph3dMapper 

### PR DESCRIPTION
**Pull request by nbaraka**
changes Glyph3d to Glyph3dMapper in order to benefit from GPU perf. 

Other benefits : 
     
- The rendering seems meaningfull with a test data. 
- no crash in 3D when changing orientation 

Disadvantages : 

- arrow are smaller in 3D

### CODE REVIEW

File: src/plugins/legacy/itkDataImage/interactors/medVtkViewItkVectorFieldInteractor.cpp [hunk 1]
```diff
@@ -178,6 +177,7 @@ void medVtkViewItkVectorFieldInteractor::setInputData(medAbstractData *data)
 
         vtkImageData *vtkImage = d->floatFilter->GetOutput();
         d->manager->SetInput(vtkImage);
+
```
Comment (mathildemerle) [inline src/plugins/legacy/itkDataImage/interactors/medVtkViewItkVectorFieldInteractor.cpp:180]: you can remove this line

File: src/plugins/legacy/itkDataImage/interactors/medVtkViewItkVectorFieldInteractor.cpp [hunk 2]
```diff
@@ -462,4 +464,20 @@ void medVtkViewItkVectorFieldInteractor::updateSlicingParam()
 
 void medVtkViewItkVectorFieldInteractor::updatePlaneVisibility()
 {
+    markMapperModified();
+}
+
+void medVtkViewItkVectorFieldInteractor::markMapperModified() {
```
Comment (mathildemerle) [inline src/plugins/legacy/itkDataImage/interactors/medVtkViewItkVectorFieldInteractor.cpp:470]: can you put the brackets under the lines, in this function?

File: src/plugins/legacy/itkDataImage/manager/vtkVectorVisuManager.cxx [hunk 3]
```diff
@@ -24,6 +24,7 @@
 #include <vtkInformation.h>
 #include <vtkLookupTable.h>
 
+
```
Comment (mathildemerle) [inline src/plugins/legacy/itkDataImage/manager/vtkVectorVisuManager.cxx:27]: to remove

File: src/plugins/legacy/itkDataImage/manager/vtkVectorVisuManager.cxx [hunk 4]
```diff
@@ -104,8 +97,7 @@ void vtkVectorVisuManager::SetInput(vtkImageData* data, vtkMatrix4x4 *matrix)
 
     this->VOI->SetInputData ( this->Input );
     this->Orienter->SetOrientationMatrix(matrix);
-    this->NormalsOrienter->SetOrientationMatrix(matrix);
-
+    this->GlyphMapper->Modified(); // a voir
```
Comment (mathildemerle) [inline src/plugins/legacy/itkDataImage/manager/vtkVectorVisuManager.cxx:100]: is it ok?

File: src/plugins/legacy/itkDataImage/manager/vtkVectorVisuManager.h [hunk 5]
```diff
@@ -27,6 +28,7 @@
 #include <vtkVectorOrienter.h>
 #include <vtkAssignAttribute.h>
 
+
```
Comment (mathildemerle) [inline src/plugins/legacy/itkDataImage/manager/vtkVectorVisuManager.h:31]: you can remove

**Comment by nbaraka** [comment]
I close this one to create a new PR. I have done a rebase on this branch to test it but I could not push it now. I will reopen the same with the needed changes. 
@mathildemerle  yes it solve #640 tbut he rendering is less smart in 3D. But It do not think that it is relevant to watch it in 3D. (but at least there is no crash)

**Pull request closed without merging**